src/components/Home/Home.tsx

```diff
@@ -1,10 +1,24 @@
-import { Layout, Page, TextStyle, Banner } from "@shopify/polaris";
+import {
+  Layout,
+  Page,
+  TextStyle,
+  Banner,
+  Form,
+  FormLayout,
+  TextField,
+  Button,
+  Card,
+} from "@shopify/polaris";
 import React, { useState } from "react";
 import { PharmacyList } from "../PharmacyList";
 import { TwitterTimelineEmbed } from "react-twitter-embed";
 
 export function Home() {
   const [showBanner, setShowBanner] = useState(true);
+  const [showPostalPrompt, setShowPostalPrompt] = useState(true);
```

Comment:
In any programming language, it is typical to prefix booleans variables. The most common prefix `is`, but other conventions include with `are`, `has`, `should`, `can`

I have found it to be an extremely helpful cue, especially in JSX.

Reply:
Changed to "shouldShow..."